src/swapDataBlock.js

```diff
@@ -0,0 +1,97 @@
+import { EditorState } from "draft-js";
+
+const swapBlocks = ({ currentKey, targetKey, editorState }) => {
+    const contentState = editorState.getCurrentContent(),
+      selectionState = editorState.getSelection(),
+      blockMap = contentState.getBlockMap(),
+      keySeq = blockMap.keySeq(),
+      getIndex = key => keySeq.findIndex(k => k === key),
+      insertBlock = (target, key, val) => {
```

Comment:
Could you initialize the variables in each line, to keep standardized with the current codebase?